addon/mixins/route.js

```diff
@@ -199,6 +200,17 @@ export default Ember.Mixin.create({
   },
 
   /**
+   Call additional functions after finding the infinityModel in the Ember data store.
+
+   @method afterInfinityModel
+   @param {Function} infinityModelPromise The resolved result of the Ember store find method. Passed in automatically.
+   @return {Ember.RSVP.Promise}
+   */
+  afterInfinityModel(infinityModelPromiseResult) {
+    return infinityModelPromiseResult;
```

Comment:
yes true! they don't have to use it, but if they DO use it, they will have to use it correctly, by returning something that implements `get('content')`, or else downstream everything breaks. BUT we don't have to require this! By implementing my suggestion above, the unnecessary requirement to return something from the hook can be avoided.